examples/torch/mtsac_MT10.py

```diff
@@ -0,0 +1,100 @@
+"""MTSAC implementation based on Metaworld. Benchmarked on MT10.
+
+https://arxiv.org/pdf/1910.10897.pdf
+"""
+import click
+from metaworld.benchmarks import MT10
+import numpy as np
+from torch import nn
+from torch.nn import functional as F
+
+from garage import wrap_experiment
+from garage.envs import GarageEnv, MultiEnvWrapper, normalize
+from garage.experiment import deterministic, LocalRunner
+from garage.replay_buffer import SimpleReplayBuffer
+from garage.sampler import LocalSampler
+from garage.torch.algos import MTSAC
+from garage.torch.policies import TanhGaussianMLPPolicy
+from garage.torch.q_functions import ContinuousMLPQFunction
+import garage.torch.utils as tu
+
+
+@click.command()
+@click.option('--seed', 'seed', type=int, default=1)
+@click.option('--use_gpu', 'use_gpu', type=bool, default=False)
+@click.option('--gpu', '_gpu', type=int, default=0)
+@wrap_experiment(snapshot_mode='none')
+def mt10_mtsac_normalize_reward(ctxt=None, seed=1, use_gpu=False, _gpu=0):
+    """Train MTSAC with MT10 environment.
+
+    Args:
+        ctxt (garage.experiment.ExperimentContext): The experiment
+            configuration used by LocalRunner to create the snapshotter.
+        seed (int): Used to seed the random number generator to produce
+            determinism.
+        use_gpu (bool): Used to enable ussage of GPU in training.
+        _gpu (int): The ID of the gpu (used on multi-gpu machines).
+
+    """
+    deterministic.set_seed(seed)
+    runner = LocalRunner(ctxt)
+    task_names = MT10.get_train_tasks().all_task_names
+    mt10_train_envs = MultiEnvWrapper([
+        normalize(GarageEnv(MT10.from_task(task_name)), normalize_reward=True)
+        for task_name in task_names
+    ],
+                                      has_one_hots=True,
+                                      has_task_ids=True,
+                                      has_name=True)
+    mt10_test_envs = [
+        GarageEnv(MT10.from_task(task_name)) for task_name in task_names
+    ]
+    policy = TanhGaussianMLPPolicy(
+        env_spec=mt10_train_envs.spec,
+        hidden_sizes=[400, 400, 400],
+        hidden_nonlinearity=nn.ReLU,
+        output_nonlinearity=None,
+        min_std=np.exp(-20.),
+        max_std=np.exp(2.),
+    )
+
+    qf1 = ContinuousMLPQFunction(env_spec=mt10_train_envs.spec,
+                                 hidden_sizes=[400, 400, 400],
+                                 hidden_nonlinearity=F.relu)
+
+    qf2 = ContinuousMLPQFunction(env_spec=mt10_train_envs.spec,
+                                 hidden_sizes=[400, 400, 400],
+                                 hidden_nonlinearity=F.relu)
+
+    replay_buffer = SimpleReplayBuffer(env_spec=mt10_train_envs.spec,
+                                       size_in_transitions=int(1e6),
+                                       time_horizon=1)
+
+    timesteps = 20000000
+    batch_size = int(150 * mt10_train_envs.num_tasks)
+    num_evaluation_points = 500
+    epochs = timesteps // batch_size
```

Comment:
to simplify the example, just pass these numbers as literals rather than doing this calculation.